tests/queries/0_stateless/02864_statistics_bug_67742.sql

```diff
@@ -0,0 +1,27 @@
+SET allow_experimental_statistics = 1;
```

Comment:
(this is an existing test, it was renamed from generic name `_bugs` to a specific name)